vidify/player/web/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""
+"""
+
+from typing import Optional
+from webbrowser import open
+from threading import Timer, Thread
+
+try:
+    import flask
+except OSError:
+    raise ImportError("Flask is not installed")
+
+from vidify.player.generic import PlayerBase
+from qtpy.QtCore import QObject, Slot, Signal, Qt
+from qtpy.QtNetwork import QTcpServer, QHostAddress, QTcpSocket
+from qtpy.QtWidgets import QVBoxLayout, QLabel
+from vidify.gui import Res, Fonts
+import sys
+import os
+
+PORT = 9999
+ROOT_URL = 'http://localhost:{}/video'.format(PORT)
+DEFAULT_VIDEO_ID = "GfKs8oNP9m8"
+class WebPlayer(PlayerBase):
+    DIRECT_URL = False
+    flask_app = None
```

Comment:
These non-constant values should go inside `__init__` rather than here (`flask_app`, `current_media` and `is_playing`). You can initialize them with `self.current_media = None`. Otherwise they're considered static members of the class, which is not what we want.